main.rb

```diff
@@ -11,36 +11,19 @@ def list_options
     7 - Exit'
 end
 
-def option(option, app) # rubocop:disable Metrics/CyclomaticComplexity
-  case option
-  when 1
-    app.list_books
-  when 2
-    app.list_people
-  when 3
-    app.create_person
-  when 4
-    app.create_book
-  when 5
-    app.create_rental
-  when 6
-    app.list_rentals
-  when 7
-    exit
-  else
-    puts 'Invalid option, please type correct number!'
-  end
+def intro
+  "Welcome to School Library App!\n\n"
 end
 
 def main
   app = App.new
 
-  puts "Welcome to School Library App!\n\n"
+  puts intro
 
   loop do
     puts list_options
     option = gets.chomp.to_i
-    option(option, app)
+    Option.option(option, app)
   end
```

Comment:
- [ ] This loop is not making the app run as expected, none of the options are responding, I will encourage you to double-check it.
![image](https://github.com/tajulafreen/School_library/assets/86200356/00051f5d-4c80-4bf0-a8ef-dc2e9dbc3b91)